composer.json

```diff
@@ -3,13 +3,13 @@
     "description": "Tool to compare two revisions of a public API to check for BC breaks",
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
-        "ext-json": "*",
         "azjezz/psl": "^2.3.1",
         "composer/composer": "^2.5.1",
         "nikic/php-parser": "^4.15.3",
         "nikolaposa/version": "^4.1.0",
         "ocramius/package-versions": "^2.7.0",
         "roave/better-reflection": "^6.5.0",
+        "seld/phar-utils": "^1.2",
```

Comment:
Really part of `require`? :thinking:

Reply:
Maybe `require-dev`?

Reply:
True, will do tomorrow, went to bed already 😴